docs/assets/templates/workflows/template-big.dockstore.yml

```diff
@@ -65,6 +65,11 @@ workflows:
     # A value of true will automatically display the latest tag updated as default.
     # A value of false will retain the default version that has been specified via the Dockstore UI.
     latestTagAsDefault: <Boolean>
+
+    # 'enableAutoDois' is a workflow-wide setting that will affect ALL branches/tags; only set this as needed in a main branch
```

Comment:
Beyond the scope of this PR, but this statement got me thinking, maybe the web service should only honor this and other workflow-level scoped settings if on the GitHub default branch.

Even with this advice, if a user reads it, it's going to be hard to do; if you create a new branch from the default branch, the .dockstore.yml is copied to the new branch.

No need to change anything here; I'll create a ticket for this idea; not sure if it's feasible.

Reply:
https://ucsc-cgl.atlassian.net/browse/SEAB-6869